rxandroidble/src/main/java/com/polidea/rxandroidble2/ConnectionSetup.java

```diff
@@ -87,8 +95,19 @@ public Builder setOperationTimeout(Timeout operationTimeout) {
             return this;
         }
 
+        /**
+         * @param connectTimeout Timeout after which the connection will be considered as broken. Eventually the connection
+         *                         will be canceled and removed from queue. Keep in mind that it will cancel the library's connection
+         *                         only and may leave Android's BLE stack in an inconsistent state.
+         * @return this builder instance
+         */
+        public Builder setConnectTimeout(Timeout connectTimeout) {
```

Comment:
what about `setConnectionTimeout` (as you have before `setOperationTimeout()`)?

Reply:
Maybe `setConnectingTimeout`? `Connection` may be understood as a whole but `Connecting` is the process of establishing a `Connection`?

Reply:
@dariuszseweryn you're right. `setConnectingTimeout` would be better as it is more accurate.